<ui version="4.0" >
 <class>OLoadDictionaryDialog</class>
 <widget class="QDialog" name="OLoadDictionaryDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>378</width>
    <height>159</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" colspan="2" >
    <widget class="QLabel" name="label_2" >
     <property name="font" >
      <font>
       <family>OStayPuft</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text" >
      <string>Archivo Diccionario</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" >
    <widget class="QPushButton" name="OLoadPushButton" >
     <property name="text" >
      <string>Cargar</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLineEdit" name="ODictionaryLineEdit" />
   </item>
   <item row="1" column="1" >
    <widget class="QToolButton" name="OGetFileToolButton" >
     <property name="text" >
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" >
    <widget class="QPushButton" name="OQuitPushButton" >
     <property name="text" >
      <string>Cancelar</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QCheckBox" name="ODictionayDefaultCheckBox" >
     <property name="font" >
      <font>
       <family>OStayPuft</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text" >
      <string>Diccionarios Por Defecto</string>
     </property>
    </widget>
   </item>
   <item rowspan="3" row="2" column="1" colspan="2" >
    <widget class="QLabel" name="OViewLabel" >
     <property name="text" >
      <string/>
     </property>
     <property name="pixmap" >
      <pixmap resource="OJohnDicDialog.qrc" >:/home/andresete/WorkSpace/OHackSuite/OIcons/JOhnICons/!.png</pixmap>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0" >
    <widget class="QLabel" name="label" >
     <property name="font" >
      <font>
       <family>OStayPuft</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text" >
      <string>Diccionarios Disponibles</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" >
    <widget class="QComboBox" name="ODictionaryComboBox" />
   </item>
  </layout>
 </widget>
 <resources>
  <include location="OJohnDicDialog.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>ODictionaryLineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>OLoadPushButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>175</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel" >
     <x>301</x>
     <y>25</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
